From e9e2ff0a4c0bcc0d4970c321df6a83d972c5761f Mon Sep 17 00:00:00 2001
From: ylevavi <yehuda.levavi@tandemg.com>
Date: Sun, 07 Nov 2021 14:56:19 +0200
Subject: [PATCH] Revert "[SW-56463] spi: reduce gaudi2 onix baud rate"

This reverts commit 8fbaa65ee0db14cf24878130ce30e823feb568dd.

Change-Id: I8f4c79d8d1c47236eabc2497185f05c3765d0161
Signed-off-by: ylevavi <yehuda.levavi@tandemg.com>
---

diff --git a/arch/gaudi2/arc/onix.c b/arch/gaudi2/arc/onix.c
index 0bd31f1..605400a 100644
--- a/arch/gaudi2/arc/onix.c
+++ b/arch/gaudi2/arc/onix.c
@@ -21,7 +21,7 @@
 
 u32 hl_spi_baudr_get(void)
 {
-	u32 factor = 4; /* Min Divider */
+	u32 factor = 2; /* Min Divider */
 
 	/* baud rate doesn't support odd number */
 	return (factor + 1) & 0xFFFE;
